### ISSUE TITLE
Spinners are silent to blind users

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1617]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

Spinners should announce that progress is happening, but say nothing right now.
